src/features/workflow/WorkflowCellControls.tsx

```diff
@@ -32,12 +34,17 @@ const WorkflowCellControls: React.FC<WorkflowCellControlsProps> = ({
     dispatch(duplicateWorkflowTransformStep(index))
   }
 
+  const onOutputClear = () => {
+    if (sessionId)
+      dispatch(removeEvent(sessionId))
+  }
+
   return (
     <div className={'group-hover:opacity-100 flex-grow-0 w-48 ml-8 opacity-0 transition-opacity relative'}>
       <div className={'flex flex-col bg-white w-48 rounded-md absolute p-5 pt-2.5'}>
         <WorkflowCellControlButton onClick={onDelete} label='Delete' icon='trashBin'/>
         <WorkflowCellControlButton onClick={onDuplicate} label='Duplicate block' icon='duplicate'/>
-        <WorkflowCellControlButton onClick={() => { dispatch(clearOutputWorkflowTransformStep(index)) }} label='Clean output' icon='circleX'/>
+        <WorkflowCellControlButton onClick={onOutputClear} label='Clean output' icon='circleX'/>
```

Comment:
ahhh, okay, i didn't understand what this was when I was reviewing before.

Based on the UI it looks like it's asking to "clean" the output of *just that cell*, while the way it is coded up is cleaning the output of every cell. 

I think we should adjust this for now. In the above, "outputClear" function, instead of clearing the events, just `console.log` saying "not yet implemented", and we should add this to the work you are doing on #313

Reply:
Yep, Will be done as part of that PR.

Reply:
okay, as long as we know it's going to be fixed soon, this can stay as is.